Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1911,6 +1914,20 @@ def vtkIterate(iterator):
         obj = iterator.GetNextItem()
 
 
+# Return gmbounds if gmbounds are different than 1.e20
+def getPlottingBounds(gmbounds, databounds, geo):
+    x1gm, x2gm, y1gm, y2gm = gmbounds[:4]
+    x1, x2, y1, y2 = databounds[:4]
+    if geo:
+        return [x1, x2, y1, y2]
+    assert (x1 < x2 and y1 < y2)
+    if not numpy.allclose([x1gm, x2gm], 1.e20):
+        x1, x2 = [x1gm, x2gm]
+    if (isinstance(y1gm, numbers.Number) and not numpy.allclose([y1gm, y2gm], 1.e20)):
+        y1, y2 = [y1gm, y2gm]
+    return [x1, x2, y1, y2]
+
+
```

Comment:
@doutriaux1 @aashish24 It ends up being the same thing. Maybe instead of using getworldcoordinates which sets zooming to be datawc or the data without bounds, we should use datawc or the full data - with bounds. What do you guys think? That might cause some incompatibilities as it changes how the package behaves.